frontend/src/__tests__/cypress/cypress/tests/mocked/modelServing/modelServingDeploy.cy.ts

```diff
@@ -314,15 +314,24 @@ describe('Model Serving Deploy Wizard', () => {
 
     modelServingGlobal.visit('test-project');
     modelServingGlobal.findDeployModelButton().click();
+    cy.url().should('include', 'ai-hub/deployments/test-project/deploy/create');
```

Comment:
This is funny cause we could technically merge this now since the tests in CI passed, but it will fail since my PR got merged. This url should be 
```suggestion
    cy.url().should('include', 'ai-hub/deployments/test-project/deploy');
```